test/integration/admin_login_test.rb

```diff
@@ -14,10 +14,10 @@ def setup
                 email: 'example@example.com',
                 role: 1)
 
-    @item1 = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
-    @category1 = item1.categories.create(name: 'Hot Beverages')
-    @item2 = Item.create(title: 'cold pressed coffee', price: 8000, description: "hipster nonsense", price: 20000)
-    @category2 = item2.categories.create(name: 'cold beverages')
+    @category1 = Category.create(name: 'Hot Beverages')
+    @category2 = Category.create(name: 'cold beverages')
+    @item1 = category1.items.create(title: 'espresso', description: "this is black gold", price: 30000)
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.